base/db/R/query.dplyr.R

```diff
@@ -132,14 +132,14 @@ workflow <- function(bety, workflow_id) {
 #' @export
 runs <- function(bety, workflow_id) {
   Workflows <- workflow(bety, workflow_id) %>%
-    dplyr::select(workflow_id, folder)
+    dplyr::select(.data$workflow_id, .data$folder)
   Ensembles <- dplyr::tbl(bety, "ensembles") %>%
-    dplyr::select(ensemble_id = id, workflow_id) %>%
-    dplyr::inner_join(Workflows, by = "workflow_id")
+    dplyr::select(.data$ensemble_id = .data$id, .data$workflow_id) %>%
```

Comment:
Creating a new column here called `ensemble_id` because it's on the left side of the expression. `.data` pronoun not needed here, and in fact, will not work.

```suggestion
    dplyr::select(ensemble_id = .data$id, .data$workflow_id) %>%
```